client/webserver/site/.stylelintrc

```diff
@@ -1,7 +1,17 @@
 {
-    "extends": "stylelint-config-standard",
+    "extends": ["stylelint-config-standard-scss"],
     "rules": {
         "at-rule-no-unknown": null,
-        "no-descending-specificity": null
+        "font-family-name-quotes": "always-unless-keyword",
+        "no-descending-specificity": null,
+        "property-no-vendor-prefix": null,
+        "selector-attribute-quotes": "never",
+        "selector-id-pattern": null,
+        "value-keyword-case": null,
+        "scss/dollar-variable-pattern": null,
```

Comment:
default pattern for these variable names is "kebab" case (`'^(-?[a-z][a-z0-9]*)(-[a-z0-9]+)*$'`) and I didn't see a good reason to go changing them all.